empyrical/stats.py

```diff
@@ -720,11 +708,11 @@ def alpha_beta_aligned(returns, factor_returns, risk_free=0.0, period=DAILY,
 
     Returns
     -------
-    float
-        Alpha.
-    float
-        Beta.
-
+    list
+        float
+            Alpha.
```

Comment:
Not sure how that happened (the tab in there). Also, I don't think I intended to keep the doc string "Returns" syntax with the `list` nomenclature, that was some WIP I had in there to remind me that a list came back with that. I've reverted this to the original "pair o' floats" style "Returns" section, and ensured that's how it is across the board.